pkg/lint/rules/chartfile.go

```diff
@@ -104,9 +105,14 @@ func validateChartYamlFormat(chartFileError error) error {
 }
 
 func validateChartName(cf *chart.Metadata) error {
+	var validNameRegex = regexp.MustCompile(`^[a-z0-9](([-a-z0-9]*)?)$`)
```

Comment:
minor: this could be a package scoped? (no need to instantiate every time the function is called)